### ISSUE TITLE
Incorrect LastPass CSV Detection

### ISSUE THREAD

**Issue by arimgibson** [padloc/padloc#732]
After trying to import my LastPass export, the file is recognized as a standard CSV file upload.

![image](https://github.com/padloc/padloc/assets/43756494/70774f1e-9356-4d2c-835d-e23d8b525f88)

I believe this is due to LastPass changing the order of headings in their CSV, to move `totp` after password instead of at the end:
Expected by Padloc
https://github.com/padloc/padloc/blob/6e3cf04fa5fc699c334a8ef85704e26a80f224eb/packages/app/src/lib/import.ts#L407-L418

Actual by LastPass Format
`url,username,password,totp,extra,name,grouping,fav`
![image](https://github.com/padloc/padloc/assets/43756494/5c2516a1-9975-40b0-b22b-4772e3bea303)

I'm currently cloning and testing if changing the expected order allows Padloc to properly identify the LastPass export. If so, I'll fork and add a PR. Assuming this size and simplicity of PR doesn't require a RFA or any other check from maintainers before PR submittal.

Thanks!!

**Comment by arimgibson** [comment]
Actually, to support legacy CSV formats, the change will instead add checking for this new format, instead of overriding the current totp check:
```typescript
 export async function isLastPass(file: File): Promise<boolean> { 
     try { 
         const data = await readFileAsText(file); 
         const headerRow = data.split("\n")[0].trim(); 
         return ( 
             headerRow === "url,username,password,extra,name,grouping,fav" || 
             headerRow === "url,username,password,extra,name,grouping,fav,totp" ||
             headerRow === "url,username,password,totp,extra,name,grouping,fav"
         ); 
     } catch (error) { 
         return false; 
     } 
 } 
```

**Comment by arimgibson** [comment]
Closed via #725
